### ISSUE TITLE
Support a larger variety of FITS images

### ISSUE THREAD

**Issue by tboch** [friendlyvo/friendlyVO#7]
Currently, the service fails for many of the FITS images I tested (cherry-picked from [FITS images attached to VizieR catalogues](http://vizier.u-strasbg.fr/viz-bin/VizieR-3?-ref=VIZ564653c8407b&-to=-2b&-from=-2&-this=-2&%2F%2Fmedia=2&-out.max=50&%2F%2FCDSportal=http%3A%2F%2Fcdsportal.u-strasbg.fr%2FStoreVizierData.html&-out.form=HTML+Table&-out.add=_r&-out.add=_RAJ%2C_DEJ&%2F%2Foutaddvalue=default&-sort=_r&-order=I&-oc.form=sexa&-nav=key%3Amedia%3D2%26HTTPPRM%3A&%2F%2Fnone=on&-sort=_r&-order=I&-media=2&-c.eq=J2000&-c.r=++2&-c.u=arcmin&-c.geom=r&-file=.&-meta.ucd=2&-meta=1&-meta.foot=1&-usenav=1&-bmark=GET))

**Comment by james-allen** [comment]
Do you know what's causing them to fail? How much of this is overlap with #6, i.e. the WCS information being in an extension HDU rather than the primary?

**Comment by james-allen** [comment]
Ok, I found this:

http://archive.stsci.edu/missions/swift_uvot/00035888/00035888001/sw00035888001uvv_sk.img

which fails because the WCS object created from the extension's header doesn't have a cd matrix. Looks like the matrix is optional in the database, and not returned anywhere. Can we just be more accepting about what we allow to pass through? What are the minimum requirements to say we have a valid WCS solution?

**Comment by tboch** [comment]
Well, as the image appears fine in Aladin and DS9, I (as a user) would expect this image to work ok in friendlyVO :)

**Comment by james-allen** [comment]
Agreed! The question is just how permissive we can go. I think the bare minimum requirements are:
- astropy.wcs.WCS(header) returns without error
- NAXIS1 and NAXIS2 are present in the header (which they always should!)
- wcs.wcs_pix2world(corners_x, corners_y, 0) returns without error

If we make these the requirements, are there any cases that may cause us problems further down the line? Particularly if there are additional types of queries to be implemented on the same data?

**Comment by james-allen** [comment]
For the record, the image I linked to does satisfy the requirements I listed.
